src/MyConnection.h

```diff
@@ -18,7 +18,8 @@ class MyConnection : boost::noncopyable {
 
 public:
 
-  MyConnection(std::string host, std::string user, std::string password,
+  MyConnection(std::string host, std::string user,
+               Rcpp::Nullable < std::string > password,
```

Comment:
This is now inconsistent - why does password use NULL and all the other parameters use ""? I'd rather keep things consistent and do all of the default values in one place.